src/diffpy/srfit/sas/prcalculator.py

```diff
@@ -92,7 +92,7 @@ def __call__(self, r):
         self._invertor.y = iq
         self._invertor.err = diq
         c, c_cov = self._invertor.invert_optimize()
-        calculate_pr = lambda x: self._invertor.pr(c, x)
```

Comment:
I think using lambda functions that are short and concise actually increase readability... Hence adding `# noqa: E731` as well.